Source/WTF/wtf/glib/SysprofAnnotator.h

```diff
@@ -0,0 +1,437 @@
+
+        case WebHTMLViewPaintStart:
+        case WebHTMLViewPaintEnd:
+            return "WebHTMLViewPaint"_s;
+
+        case BackingStoreFlushStart:
+        case BackingStoreFlushEnd:
+            return "BackingStoreFlush"_s;
+        case BuildTransactionStart:
+        case BuildTransactionEnd:
+            return "BuildTransaction"_s;
+        case SyncMessageStart:
+        case SyncMessageEnd:
+            return "SyncMessage"_s;
+        case SyncTouchEventStart:
+        case SyncTouchEventEnd:
+            return "SyncTouchEvent"_s;
+        case InitializeWebProcessStart:
+        case InitializeWebProcessEnd:
+            return "InitializeWebProcess"_s;
+        case RenderingUpdateRunLoopObserverStart:
+        case RenderingUpdateRunLoopObserverEnd:
+            return "RenderingUpdateRunLoopObserver"_s;
+        case LayerTreeFreezeStart:
+        case LayerTreeFreezeEnd:
+            return "LayerTreeFreeze"_s;
+        case FlushRemoteImageBufferStart:
+        case FlushRemoteImageBufferEnd:
+            return "FlushRemoteImageBuffer"_s;
+        case CreateInjectedBundleStart:
+        case CreateInjectedBundleEnd:
+            return "CreateInjectedBundle"_s;
+        case PaintSnapshotStart:
+        case PaintSnapshotEnd:
+            return "PaintSnapshot"_s;
+        case RenderServerSnapshotStart:
+        case RenderServerSnapshotEnd:
+            return "RenderServerSnapshot"_s;
+        case TakeSnapshotStart:
+        case TakeSnapshotEnd:
+            return "TakeSnapshot"_s;
+        case SyntheticMomentumStart:
+        case SyntheticMomentumEnd:
+            return "SyntheticMomentum"_s;
+        case SyntheticMomentumEvent:
+            return "SyntheticMomentumEvent"_s;
+        case RemoteLayerTreeScheduleRenderingUpdate:
+            return "RemoteLayerTreeScheduleRenderingUpdate"_s;
+
+        case CommitLayerTreeStart:
+        case CommitLayerTreeEnd:
+            return "CommitLayerTree"_s;
+        case ProcessLaunchStart:
+        case ProcessLaunchEnd:
+            return "ProcessLaunch"_s;
+        case InitializeSandboxStart:
+        case InitializeSandboxEnd:
+            return "InitializeSandbox"_s;
+        case WebXRCPFrameWaitStart:
+        case WebXRCPFrameWaitEnd:
+            return "WebXRCPFrameWait"_s;
+        case WebXRCPFrameStartSubmissionStart:
+        case WebXRCPFrameStartSubmissionEnd:
+            return "WebXRCPFrameStartSubmission"_s;
+        case WebXRCPFrameEndSubmissionStart:
+        case WebXRCPFrameEndSubmissionEnd:
+            return "WebXRCPFrameEndSubmission"_s;
+
+        case WakeUpAndApplyDisplayListStart:
+        case WakeUpAndApplyDisplayListEnd:
+            return "WakeUpAndApplyDisplayList"_s;
+
+        case WTFRange:
+        case JavaScriptRange:
+        case WebCoreRange:
+        case WebKitRange:
+        case WebKit2Range:
+        case UIProcessRange:
+        case GPUProcessRange:
+        default:
```

Comment:
I don't recommend using a default case here because it means nobody will add new enum values to the switch statement. If you remove the default case, then any new values will trip -Wswitch and the EWS should fail.

With the default case removed, you'll need to add a RELEASE_ASSERT_NOT_REACHED() after the switch statement to avoid tripping -Wreturn-type.